pandas/tools/util.py

```diff
@@ -61,6 +61,26 @@ def to_numeric(arg, errors='raise'):
         - If 'raise', then invalid parsing will raise an exception
         - If 'coerce', then invalid parsing will be set as NaN
         - If 'ignore', then invalid parsing will return the input
+    downcast : {None, 'integer', 'signed', 'unsigned', 'float'} , default None
+        If not None, and if the data has been successfully cast to a
```

Comment:
No, it is a requirement.  I explicitly check if the data is numeric before attempting any downcast.  Why attempt to downcast if the data is `object` for example?  Then you're essentially doing what you just tried to do (and failed).